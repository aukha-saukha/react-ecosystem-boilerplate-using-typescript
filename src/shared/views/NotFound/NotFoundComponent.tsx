import * as React from 'react';

const NotFoundComponent: React.FC<{}> = (): JSX.Element => (
  <main className="font-size-12">Four oh! Four</main>
);

export default NotFoundComponent;
