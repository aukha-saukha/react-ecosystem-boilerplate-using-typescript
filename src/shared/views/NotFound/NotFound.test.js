import { shallow } from 'enzyme';
import React from 'react';

import NotFound from './NotFoundComponent';

describe('"Not Found" component', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(<NotFound />);

    expect(wrapper).toMatchSnapshot();
  });
});
