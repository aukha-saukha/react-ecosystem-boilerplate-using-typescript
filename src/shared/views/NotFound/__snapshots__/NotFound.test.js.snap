// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`"Not Found" component Renders correctly 1`] = `
<main
  className="font-size-12"
>
  Four oh! Four
</main>
`;
